src/console_cmds.cpp

```diff
@@ -1248,7 +1248,7 @@ DEF_CONSOLE_CMD(ConReloadAI)
 		return true;
 	}
 
-	if (Company::IsHumanID(company_id)) {
+	if (Company::IsHumanID(company_id) || company_id == _local_company) {
```

Comment:
Maybe more to the question: why is `IsHumanID` not returning true, as clearly the human is in the company?

Reply:
I think the case where a human is in an AI company can only happen in offline mode after cheating (or loading a dedicated server save with an AI in first slot).
And `IsHumanID` is also used to determine if a network client can join/be moved to a company, and to prevent some disasters that would destroy AI infrastructure.
So I'll add comments.